7.11 Capstone 1 - Analytic Report and Research Proposal.ipynb

```diff
@@ -377,29 +336,30 @@
     "\n",
     "dfs = [df2014,df2015,df2016,df2017]\n",
     "deaths_total = df2014['DEATHS'] + df2015['DEATHS'] + df2016['DEATHS'] + df2017['DEATHS']\n",
-    "\n",
     "dftotals['TOTAL DEATHS'] = deaths_total\n",
-    "dftotals['Total Background Checks'] = st_totals\n",
-    "print(dftotals)\n",
+    "# print(dftotals)\n",
+    "# print(st_totals)\n",
+    "dftotals['Total Background Checks'] = st_totals['totals']\n",
```

Comment:
i assumed you didnt need duplicated of the state column so i just added the totals count

Reply:
this made the plot look wrong youd need to add another line 
``` dftotals['state'] = st_totals['state']\n ````